consultation_analyser/consultations/jinja2/base.html

```diff
@@ -40,11 +40,11 @@
 
   <main class="govuk-main-wrapper" id="main-content" role="main">
     <div class="govuk-width-container">
-      {% if app_config.show_provisional_data_warning %}
+      <!-- {% if app_config.show_provisional_data_warning %}
```

Comment:
This can probably be deleted throughout the codebase - but I can pick up in a separate PR.

Reply:
Separate PR: https://github.com/i-dot-ai/consult/pull/422 to delete - no need to do anything else in this PR.